packages/nx/src/nx-cloud/utilities/is-workspace-claimed.ts

```diff
@@ -1,15 +1,16 @@
-import { getCloudUrl } from './get-cloud-options';
+import { getCloudUrl, isNxCloudId } from './get-cloud-options';
 
-export async function isWorkspaceClaimed(nxCloudAccessToken) {
-  if (!nxCloudAccessToken) return false;
+export async function isWorkspaceClaimed(accessToken: string) {
+  if (!accessToken) return false;
 
   const apiUrl = getCloudUrl();
   try {
+    const requestData = isNxCloudId(accessToken)
```

Comment:
We should be able to tell when invoking this function whether or not it is an ID or an access token based on where we read it/created it from.

Let's address this separately.